### PR TITLE
[WIP]: Unit Tests for GrapesJsBuilderBundle

### PR DESCRIPTION
**Pull request by mabumusa1**
| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | features
| Bug fix?                               | no
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:

Unit tests for the GrapesJsBuilderBundle 

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2.

### CODE REVIEW

File: .ddev/commands/web/live [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+## #ddev-generated
+## Description: Run ddev-live inside the web container
+## Usage: live [flags] [args]
+## Example: "ddev live auth" or "ddev live list sites" or "ddev live --version"
+
+ddev-live $@
```
Comment (RCheesley) [inline .ddev/commands/web/live]: Should this be committed? I am not sure that it relates to this PR?

Comment (mabumusa1) [reply .ddev/commands/web/live]: I removed this file, I think we should look into `.gitignore` rules as this file was auto-generated by ddev and should be ignored

**Comment by RCheesley** [comment]
@mabumusa1 is this still WIP or do you need help with addressing the failing tests?

**Comment by RCheesley** [comment]
@mabumusa1 were you able to look into why PHPSTAN is failing on this PR?

**Comment by RCheesley** [comment]
Closing this one as we have had tests written in another PR - thank you for your help with this @mabumusa1 !

**Pull request closed without merging**